src/request-logger.js

```diff
@@ -0,0 +1,31 @@
+exports = module.exports = function (logger, data) {
+  logger = logger || console;
+  data = data || {};
+  return attachSuperagentLogger.bind(null, logger, data);
+};
+
+function attachSuperagentLogger (logger, data, req) {
+  let start = new Date().getTime();
+  let timestamp = new Date().toISOString();
+  let method = req.method;
+
+  logger.info('%s >>> %s %s \n Body: %s',
```

Comment:
So the logger that runs inside of streaming client doesn't support lots of args. The method signature of that logger looks like this: https://github.com/purecloudlabs/genesys-cloud-client-logger/blob/master/src/logger.ts#L72

We also have a lot of inconsistencies with how we log things and it causes issue. We try and log a sumo-searchable string as the message then put the details as a json object:
```
      this.logger.info('Streaming client disconnected.', { channelId });
```
 
So in this case, I would do something like:
```
logger.info(`request: ${method.toUppercase} ${req.url}`, { timestamp, data });
```

Also, would these logs fit into `debug` level? If not, we should make the request logging an opt-in per request thing.